src/client/ui/github-configuration.hbs

```diff
@@ -111,6 +113,7 @@
 
     </section>
     <script src='/public/js/github-configuration.js' nonce='{{nonce}}'></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/aui/9.4.0/aui/aui-prototyping.min.js" integrity="sha512-iUshpSgJ26ryfeS5STpopGIsONtcP9nlIgLU24C8jwGPjsSTO5vqepMFU/KWc3fTIbNhbSTS/arJAGSnMPqQeA==" crossorigin="anonymous" referrerpolicy="no-referrer" nonce='{{nonce}}'></script>
```

Comment:
Do we need the JS for anything?  Are we using the AUI components?

Reply:
It's for the loader. The animation for that is handled by JS in AUI, not by the CSS.

Reply:
:| Well that sucks.  Maybe something to bring up with the AUI team?  Seems like something that's really simple to accomplish in css...

Reply:
https://github.com/atlassian/github-for-jira/pull/625#discussion_r692646954

Do we want to pull in the latest version? Wouldn't we want to have a little more control over specifying the version and manually update it like we would if it were an npm package?

Reply:
I was doing it in CSS. I can revert back to what I had but I'll need the same script on the jira-configuration page when I go to do the dropdown for the table (from what I looked at all animations are JS based).

Reply:
This is what I had before switching to the AUI component:
```
.getConfiguration__loader {
    animation: spin 2s linear infinite;
    border: 2px solid #f3f3f3;
    border-radius: 50%;
    border-top: 2px solid #42526e;
    height: 1.8em;
    width: 1.8em;
}
```

Reply:
It is weird that such simple animations **aren't** handled by css in AUI.

Reply:
I'm asking the aui channel about this.  I think it's because it's injecting an SVG into it depending on the size, which is kinda silly.  Would make for a great replacement with web components :P